dipy/denoise/localpca.py

```diff
@@ -133,4 +152,5 @@ def localpca(arr, sigma, patch_radius=2, tau_factor=2.3, out_dtype=None):
 
     denoised_arr = thetax / theta
     denoised_arr.clip(min=0, out=denoised_arr)
+    denoised_arr[~mask] = 0
```

Comment:
That makes sense. Might be worth being explicit about this in the docstring. Some users might be surprised to see non-brain parts of the image (e.g., skull) disappear.